memp2p/message.go

```diff
@@ -4,10 +4,10 @@ import (
 	"encoding/binary"
 
 	"github.com/ElrondNetwork/elrond-go-core/core"
-	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/ElrondNetwork/elrond-go-p2p/common"
 )
 
-var _ p2p.MessageP2P = (*message)(nil)
+var _ common.MessageP2P = (*message)(nil)
```

Comment:
I think we can drom the p2p/memp2p package, I think it is not used anymore.
Initially was supposed to be a drop-in replacement for the netMessenger. The result was a less optimal implementation

Reply:
removed it